internal/stubserver/stubserver.go

```diff
@@ -85,7 +85,10 @@ func (ss *StubServer) Start(sopts []grpc.ServerOption, dopts ...grpc.DialOption)
 	if err := ss.StartServer(sopts...); err != nil {
 		return err
 	}
-	return ss.StartClient(dopts...)
+	if err := ss.StartClient(dopts...); err != nil {
+		return err
+	}
+	return nil
```

Comment:
Why is this change required? The one-line previously was doing the same thing.

Reply:
my fault, this addition is irrelevant here